src/status.ts

```diff
@@ -18,6 +18,13 @@ export class MirrordStatus {
     }
 
     draw() {
+        const showStatusBar = vscode.workspace.getConfiguration('mirrord').get('showStatusBarButton', true);
+        
+        if (!showStatusBar) {
+            this.statusBar.hide();
```

Comment:
I wonder if you checked if `return` without `hide` suffices, since we explicitly call `show` below?

Reply:
It does. removed it.